src/Adapter/Module/Configuration/ModuleSelfConfigurator.php

```diff
@@ -67,7 +67,7 @@ class ModuleSelfConfigurator
     protected $defaultConfigFile = 'self_config.yml';
 
     /**
-     * @var ModuleRepository
+     * @var ModuleRepositoryInterface
      */
     protected $moduleRepository;
```

Comment:
we can also replace the annotation by a type int may be ?

Reply:
Would be great indeed, but only from PHP 7.4 :(